### ISSUE TITLE
Accessibility - keyboard shortcut for saving

### ISSUE THREAD

**Issue by MartinMuzatko** [strapi/strapi#1995]
**What is the expected behavior?**

Hello!

It would be nice to be able to save pages/models via <kbd>CTRL</kbd>+<kbd>S</kbd>.
This allows for power users to quickly edit and save pages.

This can be implemented with the document-wide eventhandler.

```js
document.addEventListener("keydown", event => {
  if (event.ctrlKey && event.keyCode == 83) {
    event.preventDefault()
    // save object
  }
}, false);
```

**Comment by MartinMuzatko** [comment]
Agreed. It does not happen that often that you want to save a webpage, and even more seldom save a webpage offline of an only-online working admin panel. but you never know.
![image](https://user-images.githubusercontent.com/2950505/46280932-89466800-c56d-11e8-811f-4d06405a89cc.png)

Atlassian does it in almost every tool (jira, confluence) and many more software. It is just a little extra. But if you are so fond of the keyboard, you will more likely use the terminal commands anyway. Even though the UI is easier, the experience is so great that I never resorted to the terminal cli commands ;)

**Comment by jrgochan** [comment]
> I'm keeping the suggestion in mind, we should do a complete keyboard shortcuts implementation. I'm closing the issue to keep the repository as clear as possible. Feel free to comment and ping me if needed.

++support_this_idea

**Comment by larswittenberg** [comment]
What happened to this?
